pyproject.toml

```diff
@@ -3,14 +3,15 @@ requires = ["setuptools >= 61.0", "autosemver"]
 # build-backend = "setuptools.build_meta"
 
 [project]
-requires-python = "==3.9.*"
+requires-python = ">3.9.0"
```

Comment:
Do we want to support older Python versions - the demand might never materialise? Until we're and testing different versions with e.g. `tox`, I'd probably prefer to insist on `>=3.11` or even `>=3.12`.

Reply:
I'm happy with that (the slightly former, more conservative) unless @albags  has a different take